beacon_ax: 0.05885
beacon_ay: -0.1182

beacon_bx: 2.935
beacon_by: -0.09771

beacon_cx: 1.508
beacon_cy: 2.09646

odom_freq: 100
# initial pose
initial_x: 1.5
initial_y: 1.0
initial_theta: 90

# differential drive model const.
predict_cov_a1: 1.5
predict_cov_a2: 2.5
predict_cov_a3: 1.5
predict_cov_a4: 2.5

# omnidrive model const.
#predict_const_x: 10
predict_const_x: 10
#predict_const_y: 10
predict_const_y: 10
predict_const_theta: 10
#predict_const_theta: 10
# predict_const_x: 5
# predict_const_y: 5
# # predict_const_theta: 5
# predict_const_theta: 1.5

# measurement covariance
update_cov_1: 0.015
update_cov_2: 0.015
update_cov_3: 0.015
#update_cov_1: 0.015
#update_cov_2: 0.015
#update_cov_3: 0.015

# update param=
mini_likelihood: -10000.0
mini_likelihood_update: 0.45
#mini_likelihood_update: 0.65
max_obstacle_distance: 0.5
